### ISSUE TITLE
Seems some things are missing

### ISSUE THREAD

**Issue by zapek** [temisu/ancient_format_decompressor#3]
Here's a build attempte after a fresh checkout:

```
$ make
clang++ -Os -Wall -Wsign-compare -Wshorten-64-to-32 -Wno-error=multichar -Wno-multichar -Isrc -std=c++14 -fno-rtti -o Buffer.o -c src/Buffer.cpp
make: *** No rule to make target 'Common.o', needed by 'ancient'.  Stop.
```

Ok so let's remove Common.o from the Makefile.

```
In file included from src/LHLBDecompressor.cpp:7:
src/DynamicHuffmanDecoder.hpp:19:5: error: no member named 'memset' in the global namespace; did you mean 'wmemset'?
                ::memset(_nodes,0xff,sizeof(_nodes));

```

Well, let's get rid of that too.

`LZBSDecompressor.o:(.text+0x2b8): undefined reference to `rotateBits(unsigned int, unsigned int)'`

Seems there's some stuff missing.

**Comment by temisu** [comment]
fixed by 5a104aeda9aed500a21b81119602ee48daee14c8

**Comment by temisu** [comment]
thanks for letting me know
